mmv1/products/alloydb/Cluster.yaml

```diff
@@ -497,6 +497,55 @@ properties:
         description: |
           Name of the primary cluster must be in the format
           'projects/{project}/locations/{location}/clusters/{cluster_id}'
+  - !ruby/object:Api::Type::NestedObject
+    name: 'maintenanceUpdatePolicy'
+    description: | 
+      MaintenanceUpdatePolicy defines the policy for system updates.
+    properties:
+      - !ruby/object:Api::Type::Array
+        name: 'maintenanceWindows'
+        description: |
+          Preferred windows to perform maintenance. Currently limited to 1.
+        item_type: !ruby/object:Api::Type::NestedObject
+          name: 'maintenanceWindow'
+          description: |
+            specifies a preferred day and time for maintenance.
+          properties:
+            - !ruby/object:Api::Type::Enum
+              name: 'day'
+              description: |
+                Preferred day of the week for maintenance, e.g. MONDAY, TUESDAY, etc.
+              values:
+                - :MONDAY
+                - :TUESDAY
+                - :WEDNESDAY
+                - :THURSDAY
+                - :FRIDAY
+                - :SATURDAY
+                - :SUNDAY
+            - !ruby/object:Api::Type::NestedObject
+              name: 'startTime'
+              custom_expand: 'templates/terraform/custom_expand/alloydb_cluster_input_maintenance_window_start_time_expand.go.erb'
+              custom_flatten: 'templates/terraform/custom_flatten/alloydb_cluster_input_maintenance_window_start_time_flatten.go.erb'
+              description: |
+                Preferred time to start the maintenance operation on the specified day. Maintenance will start within 1 hour of this time.
+              properties:
+                - !ruby/object:Api::Type::Integer
+                  name: hours
```

Comment:
Should these properties be marked as required? Does it make sense to specify hours but not minutes, etc?

Reply:
No, it will not make the entire maintenance_update_policy or maintenance_windows required.

You can leave minutes or seconds not required